web/src/test/java/org/springframework/security/web/util/matcher/IpAddressMatcherTests.java

```diff
@@ -153,4 +153,18 @@ public void constructorWhenRequiredAddressIsEmptyThenThrowsIllegalArgumentExcept
 			.withMessage("ipAddress cannot be empty");
 	}
 
+	// gh-16795
+	@Test
+	public void toStringWhenCidrIsProvidedThenReturnsIpAddressWithCidr() {
+		IpAddressMatcher matcher = new IpAddressMatcher("192.168.1.0/24");
+		assertThat(matcher.toString()).hasToString("IpAddressMatcher[192.168.1.0/24]");
```

Comment:
It looks like the tests don't match output of the method. Can you double-check please?